index.html

```diff
@@ -529,28 +529,50 @@ <h2>
         Examples
       </h2>
       <p>
-        This example shows the current screen orientation to the console every
-        time the screen orientation state changes.
+        In this example for a mobile phone, clicking the landscape button makes
+        a request to go fullscreen and then lock to landscape orientation. Once
+        it is in landscape, the unlock button will take the screen orientation
+        back to default (portrait) or the portrait button will also change it
+        back to portrait.
+      </p>
+      <p>
+        The log shows the change in orientation type and angle, and whether the
+        event type is load or change.
       </p>
       <pre class='example html'>
 &lt;script&gt;
-  var show = function() {
-     console.log("Orientation type is " + screen.orientation.type);
-     console.log("Orientation angle is " + screen.orientation.angle);
-  }
+async function lockLandscape(){
+  await document.documentElement.requestFullscreen() 
+  await screen.orientation.lock('landscape');
+}
+
+async function unlock() {
+  await screen.orientation.unlock();
+}
+
+async function portrait(){
+  await screen.orientation.lock('portrait');
```

Comment:
Nit: here you need the full screen check... otherwise, this will fail if the button is just pressed.

Reply:
I have added these in, just wanted to explain the reason I didn't put it in.  On my phone once you have locked to landscape you can only stay in full screen.  If you try to exit full screen it just takes you back to portrait.  So, for this specific example, where we are locking to landscape and then unlocking, we don't need the check.  Other phones might be different of course, so probably better to put it in! 😊

Reply:
Sure! Let's chat about this tomorow.